.env.example

```diff
@@ -1,6 +1,8 @@
+JAHIA_DEPLOY_METHOD=curl
+
+# specify variables below for JAHIA_DEPLOY_METHOD=curl deployment
```

Comment:
Any reasons to change the order of the env file here?

Reply:
based on this https://github.com/Jahia/javascript-components/blob/master/packages/scripts/deploy.js, only certain envs are used depending on which deploy method is being used which wasn't obvious to me at first.

I tried to organize it to make it more clearer which envs are used for certain deploy methods.

Reply:
gotcha, thanks for the clarification!